src/CR.cc

```diff
@@ -293,18 +295,14 @@ void checkSpanForCRs(detection::Footprint *extras, // Extra spans get added to t
  * Find the sum of the pixels in a Footprint
  */
 template <typename ImageT>
-class CountsInCR : public detection::FootprintFunctor<ImageT> {
+class CountsInCR {
 public:
-    CountsInCR(ImageT const& mimage, double const bkgd) :
-        detection::FootprintFunctor<ImageT>(mimage),
+    CountsInCR(double const bkgd) :
         _bkgd(bkgd),
         _sum(0.0) {}
 
-    // method called for each pixel by apply()
-    void operator()(typename ImageT::xy_locator loc, // locator pointing at the pixel
-                    int, int
-                   ) {
-        _sum += *loc - _bkgd;
+    void operator()(geom::Point2I const & point, ImageT const & val){
+        _sum += val - _bkgd;
     }
```

Comment:
anything that makes use of the applyFunctor method must have a call signature of a Point2I as its first argument. All other arguments are optional and depend only on how applyFunctor is called.